libkbfs/folder_branch_ops.go

```diff
@@ -523,7 +522,31 @@ func (fbo *folderBranchOps) setHeadLocked(ctx context.Context,
 			return nil
 		}
 
-		oldHandle = fbo.head.GetTlfHandle()
+		oldHandle := fbo.head.GetTlfHandle()
+		newHandle := md.GetTlfHandle()
+
+		resolvesTo, partialResolvedOldHandle, err :=
+			oldHandle.ResolvesTo(
```

Comment:
I think the TODO will eventually handle it, although the same problem will come up when we do checking in `MDOps`, since the MD's TlfHandle is bare only.